zipline/assets/asset_writer.py

```diff
@@ -563,3 +560,9 @@ def _load_data(self, equities, futures, exchanges, root_symbols):
             exchanges=exchanges_output,
             root_symbols=root_symbols_output,
         )
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc_info):
+        pass  # Shouldn't dispose of the engine explicitly
```

Comment:
Do you think there's still a benefit to this becoming a context manager if it hasn't nothing to clean up?